core/src/main/scala/com/karumi/shot/screenshots/ScreenshotsComparator.scala

```diff
@@ -51,11 +58,14 @@ class ScreenshotsComparator {
       val newScreenshotPixels = newScreenshot.pixels
 
       val differentPixels =
-        oldScreenshotPixels.zip(newScreenshotPixels).filter { case (a, b) => a != b }
+        oldScreenshotPixels
+          .zip(newScreenshotPixels)
+          .filter { case (a, b) => !isRGBSimilar(a, b, colorTolerance) }
+
```

Comment:
Even when our specific scenario is related to colors. I'm wondering if we should rename ``colorTolerance`` param to something more specific like ``absoluteTolerancePerChannel`` 

As this has also a huge impact in the usage of tolerance as a parameter. I'm wondering if we could update our docs with an example to show what adding this new tolerance means. Because if we add a tolerance value for ``absoluteTolerancePerChannel`` this means, the ``tolerance`` value will be used differently and this is not trivial to understand at all for a user without experience on image comparisons.